From 959d289bb4547155ae3cee5dee0f6afb686fd0be Mon Sep 17 00:00:00 2001
From: Anurag <anurag.gopinath@broadcom.com>
Date: Tue, 12 Nov 2013 13:46:56 +0530
Subject: [PATCH 0842/1532] Fix : GCC445 compilation

For inline assembly if clobberlist is defined and is empty
gcc445 toolchains errors out. So removing the empty clobberlist
definition.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/netlogic/lib/syslib/include/nlm_hal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/netlogic/lib/syslib/include/nlm_hal.h b/arch/mips/netlogic/lib/syslib/include/nlm_hal.h
index adb6092..451e0df 100644
--- a/arch/mips/netlogic/lib/syslib/include/nlm_hal.h
+++ b/arch/mips/netlogic/lib/syslib/include/nlm_hal.h
@@ -224,7 +224,7 @@ inline static u32 xlp9xx_get_node_id(void)
                 "   srl   %0, 7        \n" \
                 "   and   %0, 7        \n" \
                 " .set pop             \n" \
-                : "=r" (node):: );
+                : "=r" (node): );
 
         return node;
 }
-- 
1.9.1

